lib/commons/text/native-text-methods.js

```diff
@@ -16,15 +16,14 @@ const nativeTextMethods = {
    * @param {VirtualNode} element
    * @return {String} value text
    */
-  valueText: ({ actualNode }) => actualNode.value || '',
+  valueText: vNode => vNode.props.value || '',
 
   /**
    * Return default value of a button
    * @param {VirtualNode} element
    * @return {String} default button text
    */
-  buttonDefaultText: ({ actualNode }) =>
-    defaultButtonValues[actualNode.type] || '',
+  buttonDefaultText: vNode => defaultButtonValues[vNode.attr('type')] || '',
```

Comment:
.attr is case sensitive:

```suggestion
  buttonDefaultText: vNode => defaultButtonValues[vNode.props.type] || '',
```

Let's make sure we have a test for this too.